src/App.js

```diff
@@ -14,11 +15,79 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: [],
+  }
+
+  clear = () => {
+    const goodsList = document.querySelector('.goods');
+
+    [...goodsList.children].forEach((good) => {
+      good.classList.remove('active');
+    });
+
+    this.setState({ selected: [] });
+  }
+
+  add = ({ target }) => {
+    const listTarget = target.closest('li');
+    const newGood = listTarget.firstChild.textContent;
+
+    listTarget.classList.toggle('active');
```

Comment:
don't use target when you can(here you can avoid this) and classList, add a condition into className and add class active when you need